examples/complex-chart/app.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
I'm not sure if we still need the header on every file, as @ibgreen removed it from the contribution guidelines, need confirmation